INSTALL.md

```diff
@@ -1,3 +1,35 @@
+# Preliminaries for Ubuntu 16.04
+Installation for other distros may be slightly different.
+
+## Packages
+
+```
+sudo apt-get update
+sudo apt-get install build-essential cmake libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev libpcsclite-dev
+```
+
+## OpenCV
+
+Download the OpenCV source archive and unpack it. In source folder run following:
```

Comment:
Let's add a link to the source archive or at least to the releases page.

Reply:
Added link to releases page.